frontend/src/tests/UsernameInput.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { it, describe, beforeEach, afterEach, expect, vi } from 'vitest';
+import { flushPromises, mount } from '@vue/test-utils';
+import { UserType } from 'fam-app-acsctl-api';
+import { AppActlApiService } from '@/services/ApiServiceFactory';
+import UserNameInput from '@/components/grantaccess/form/UserNameInput.vue';
+import { isLoading, setLoadingState } from '@/store/LoadingState';
+import type { VueWrapper } from '@vue/test-utils/dist/vueWrapper';
+import type { DOMWrapper } from '@vue/test-utils/dist/domWrapper';
+import type { AxiosRequestHeaders, AxiosResponse } from 'axios';
+import { fixJsdomCssErr } from '@/tests/common/fixJsdomCssErr';
+
+fixJsdomCssErr();
+
+const userInputMock = (isUserFound: boolean): AxiosResponse => {
+    if (isUserFound) {
+        return {
+            data: {
+                firstName: 'Name',
+                found: true,
+                lastName: 'LastName',
+                userId: 'userId',
+            },
+            status: 200,
+            statusText: 'Ok',
+            headers: {},
+            config: {
+                headers: {} as AxiosRequestHeaders,
+            },
+        };
+    } else {
+        return {
+            data: {
+                found: false,
+                userId: 'userId',
+            },
+            status: 200,
+            statusText: 'Ok',
+            headers: {},
+            config: {
+                headers: {} as AxiosRequestHeaders,
+            },
+        };
+    }
+};
+
+describe('UserNameInput', () => {
+    let wrapper: VueWrapper;
+    let emitSetVerifyResult: unknown[][] | undefined;
```

Comment:
we don't need to declare it here, cause it's not shared between test cases, and we declare it in each test case already,  feel we can remove it, thanks!

Reply:
Do you mean that I should declare the emit emitSetVerifyResult on the two different tests?

Reply:
yes, just remove from there, because we do `emitSetVerifyResult = wrapper.emitted('setVerifyResult');` in the test cases, and this is not sharable.

Reply:
did not know that, thanks! I'll remove it